addons/wp/functions/Modules/fnc_moduleAssault.sqf

```diff
@@ -0,0 +1,95 @@
+#include "script_component.hpp"
+/*
+ * Author: jokoho482
+ * TODO
+ *
+ * Arguments:
+ * TODO
+ *
+ * Return Value:
+ * TODO
+ *
+ * Example:
+ * TODO
+ *
+ * Public: No
+*/
+params [["_mode", "", [""]], ["_input", [], [[]]]];
+
+switch (_mode) do {
+    // Default object init
+    case "init": {
+        if (is3DEN) exitWith {};
+        _input params [["_logic", objNull, [objNull]], ["_isActivated", true, [true]], ["_isCuratorPlaced", false, [true]]];
+        if !(_isActivated && local _logic) exitWith {};
+        if (_isCuratorPlaced) then {
+            //--- Get unit under cursor
+            GET_CURATOR_GRP_UNDER_CURSOR(_group);
+
+            if (isNull _group) then {
+                private _groups = allGroups;
```

Comment:
For the sake of the end-user this should be a more smart formula. 

I suggest: 
1. Remove all groups without units  (it will sometimes display essentially _dead_ groups) 
2. Sort units from closest to logic to furthest. 

This way, it is easier to use without memorising the name of all groups currently in play :P